modules/data.land/inst/IC_prep_anchorSites.R

```diff
@@ -0,0 +1,117 @@
+#read settings.
```

Comment:
similar to other script, need better overall documentation

Reply:
Fixed.